setup.py

```diff
@@ -53,7 +53,10 @@
     "jsonschema~=3.2.0",
     "importlib_metadata;python_version<'3.8'",
     "azure-iot-device~=2.11",
+    "tomli",
```

Comment:
It's good practice to constrain dependency so we stay within the major version range we tested with. The concern is a new major version makes a breaking change.